<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionDigital_phosphor"/>
    <addaction name="actionAddMarker"/>
    <addaction name="actionDocks"/>
   </widget>
   <widget class="QMenu" name="menuOscilloscope">
    <property name="title">
     <string>&amp;Oscilloscope</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionSampling"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
   <addaction name="menuView"/>
   <addaction name="menuOscilloscope"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSampling"/>
   <addaction name="separator"/>
   <addaction name="actionDigital_phosphor"/>
   <addaction name="separator"/>
   <addaction name="actionAddMarker"/>
   <addaction name="actionRemoveMarker"/>
   <addaction name="separator"/>
   <addaction name="actionReport_an_issue"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open layout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save layout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDigital_phosphor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Digital phosphor</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sampling</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionAddMarker">
   <property name="text">
    <string>Add marker/zoom view</string>
   </property>
  </action>
  <action name="actionDocks">
   <property name="text">
    <string>Docks</string>
   </property>
  </action>
  <action name="actionReport_an_issue">
   <property name="text">
    <string>Report an issue</string>
   </property>
  </action>
  <action name="actionRemoveMarker">
   <property name="text">
    <string>Remove Marker</string>
   </property>
   <property name="toolTip">
    <string>Remove current marker and zoomview</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
